libs/sdk-ui-ext/src/notificationsPanel/data/useNotificationFiltersDetail.tsx

```diff
@@ -73,12 +88,17 @@ export function useNotificationsFilterDetail(notification: IAlertNotification) {
                     })
                     .filter(Boolean) as ObjRef[];
 
-                const labels = await backend
-                    .workspace(workspaceId)
-                    .attributes()
-                    .getAttributeDisplayForms(filterDisplayFormsRefs);
+                const dashboardId = automation?.dashboard;
+
+                const dashboardPromise = dashboardId
+                    ? fetchDashboard(backend, workspaceId, dashboardId)
```

Comment:
I'm wondering if we don't already have the info about dashboard in the application state somewhere, could you please check? The same maybe for other fetch*.